### DISCUSSION TITLE
Inquiry about YOLO26 vs YOLO11 Performance

### DISCUSSION THREAD

**Discussion by Brayan532** [ultralytics/ultralytics#22371] [Q&A]
Dear Ultralytics Team,

First of all, thank you for introducing the new version of YOLO, i.e. YOLO26. I truly appreciate the continuous improvements your team brings to the model.

I was wondering about the differences between YOLO26 and YOLO11 in terms of mAP50-95 and latency. Based on my observations, the performance differences seem quite narrow despite the updates introduced in YOLO26. It would be helpful if YOLO11 could be included in the comparison figure alongside other versions for clearer context.

Could you please clarify whether there are key differences or improvements that I might have overlooked?

Warm regards,
Brayan

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Brayan532, thank you for your interest in Ultralytics 🚀! This is an automated response — an Ultralytics engineer will follow up soon to assist further.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question (including performance comparisons like YOLO26 vs YOLO11), please share more details to help the team contextualize the results:
-Your task and dataset (detection/segmentation/pose, class counts)
-Hardware/runtime (GPU/CPU, batch size, image size)
-Metrics you’re comparing (mAP50-95, latency settings), plus any logs or plots
-And verify you’re following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Your feedback about including YOLO11 alongside YOLO26 in comparison figures is appreciated 🙏. Please share any specific benchmarks or plots you have, and we’ll make sure the right folks see it.

**Reply by glenn-jocher** [reply]
Thanks Brayan—great question, and credit goes to the YOLO community and Ultralytics team. YOLO11 remains the recommended stable baseline; YOLO26 is still R&D, targeting smaller models, slightly higher accuracy, lower memory, and a native end‑to‑end pipeline that improves CPU/edge throughput and simplifies deployment. For context, published YOLO11 baselines on COCO val2017 @640 are YOLO11x 54.7 mAP with 11.3 ms T4 TensorRT and YOLO11n at 1.5 ms; see the YOLO11 vs YOLOv8 comparison and the YOLO11 model page for details.

Good call on the chart—adding YOLO11 alongside YOLO26 would help; we’ve noted this for the docs. If you want a like‑for‑like check on your setup, evaluate mAP with `yolo val` and measure latency using Benchmark mode; guidance is in the Benchmark mode docs.

- See YOLO11 numbers: [YOLO11 vs YOLOv8: Detailed Comparison](https://docs.ultralytics.com/compare/yolo11-vs-yolov8/)  
- Model details: [YOLO11 model page](https://docs.ultralytics.com/models/yolo11/)  
- How to measure latency: [Benchmark mode](https://docs.ultralytics.com/modes/benchmark/)
